app/code/Magento/Store/Model/Config/Placeholder.php

```diff
@@ -164,4 +206,26 @@ protected function _setValue(array &$container, $path, $value)
         }
         $currentPointer = $value;
     }
+
+    /**
+     * Set array value by path
+     *
+     * @param array &$container
+     * @param array $path
+     * @param string $value
+     * @return void
+     */
+    private function _setCurrentPathValue(array &$container, array $path, $value)
```

Comment:
Please avoid using underscores at the beginning of the private/protected method's name since this is the legacy approach according to the PSR-2 Coding Style Guide:
```
Property names SHOULD NOT be prefixed with a single underscore to indicate protected or private visibility.
```

Reply:
Removed